src/psycop_model_training/utils/utils.py

```diff
@@ -228,10 +228,21 @@ def bin_continuous_data(
         else:
             continue
 
-    # Drop any category in the series where the bin has fewer than 5 observations
-    series = series[series.groupby(series).transform("count") >= min_n_in_bin]
+    df = pd.DataFrame(
+        {
+            "series": series,
+            "bin": pd.cut(
+                series,
+                bins=bins,
+                labels=labels,
+                duplicates="drop",
+                include_lowest=True,
+            ),
+        },
+    )
 
-    return pd.cut(series, bins=bins, labels=labels, duplicates="drop")
+    # Drop any category in the series where the bin has fewer than 5 observations
+    return df.bin[df.groupby("bin")["series"].transform("size") >= min_n_in_bin]
```

Comment:
```suggestion
    bins_with_sufficient_n = df.groupby("bin")["series"].transform("size") >= min_n_in_bin
    return df.bin[bins_with_sufficient_n]
```

Splitting this out into a variable makes it **much** easier to read :-)